controller/src/beerocks/master/tasks/dynamic_channel_selection_task.cpp

```diff
@@ -0,0 +1,131 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2016-2019 Intel Corporation
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include "dynamic_channel_selection_task.h"
+
+#include "../son_actions.h"
+
+#include <bcl/beerocks_defines.h>
+#include <bcl/network/network_utils.h>
+#include <easylogging++.h>
+
+using namespace beerocks;
+using namespace net;
+using namespace son;
```

Comment:
Adding `using namespace` globally is bad practice.
Namespaces were created for a reason, one of which is to encapsulate different types and functions.
Just use the namespaces explicitly where it's needed (e.g. `beerocks::...` or `beerocks::net::...`)

Reply:
done